pypicloud/__init__.py

```diff
@@ -143,6 +143,16 @@ def run_postfork_hooks():
     config.registry.fallback = fallback_mode
     config.registry.always_show_upstream = always_show_upstream
 
+    stream_files = settings.get("pypi.stream_files")
+    if stream_files is None:
+      stream_files = False
+    else:
+      stream_files = asbool(stream_files)
+    config.registry.stream_files = stream_files
+
+    max_age = int(settings.get("pypi.package_max_age", 0))
+    config.registry.max_age = max_age
```

Comment:
Oh forgot: small request to change this to `package_max_age` to make it clearer that it's the max age specifically for packages.